cmd/experimental/kjobctl/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
How is this related to the PR? Can we move to a separete one?

Reply:
When I running `make artifacts` I have an error that `LICENSE` file not found on `multiplatform-build.sh`.

Reply:
Can we move it to a preparatory PR?

Reply:
#2819